falcon/__init__.py

```diff
@@ -363,7 +363,7 @@
 from falcon.errors import HTTPNotAcceptable
 from falcon.errors import HTTPNotFound
 from falcon.errors import HTTPNotImplemented
-from falcon.errors import HTTPPayloadTooLarge
+from falcon.errors import HTTPContentTooLarge
```

Comment:
@vytas7 do you think we should keep the old class as deprecated?

If so the new one should be a subclass of `HTTPPayloadTooLarge` I guess.

Honestly not sure.

Edit: looking at the changelog, a similar thing was done tor 1.4->2.0. let's use the same approach!

Reply:
I definitely think we ought to keep old classes as deprecated, even if 4.0 is a major release, since we have not given the users any deprecation window.